filelog.go

```diff
@@ -62,26 +98,26 @@ func (w *FileLogWriter) Close() {
 //   [%D %T] [%L] (%S) %M
 func NewFileLogWriter(fname string, rotate bool) *FileLogWriter {
 	w := &FileLogWriter{
-		rec:      make(chan *LogRecord, LogBufferLength),
-		rot:      make(chan bool),
-		completed: make(chan int),
-		filename: fname,
-		format:   "[%D %T] [%L] (%S) %M",
-		rotate:   rotate,
-	}
-
-	// If the directory doesn't exist, attempt to create it
-	logDir := path.Dir(fname)
-	err := os.MkdirAll(logDir, os.ModeDir | os.ModePerm)	
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "FileLogWriter(%q): %s\n", w.filename, err)
-		return nil
+		rec:                 make(chan *LogRecord, LogBufferLength),
+		rot:                 make(chan bool),
+		completed:           make(chan int),
+		filename:            fname,
+		format:              "[%D %T] [%L] (%S) %M",
+		rotate:              rotate,
+		rotateIntegerSuffix: false,
```

Comment:
We try to take a very light hand on formatting with the 3rd party libraries. It makes pulling in upstream changes very difficult. The upstream for this project isn't very active so I don't think it's that big of deal, but generally speaking we should be avoiding formatting only changes in forks.